pkg/serviceimport/map.go

```diff
@@ -195,14 +197,14 @@ func (m *Map) Remove(serviceImport *mcsv1a1.ServiceImport) {
 }
 
 func getServiceWeightFrom(si *mcsv1a1.ServiceImport, forClusterName string) int64 {
-	weightKey := "load-balancer.submariner.io/weight/" + forClusterName
+	weightKey := lhconstants.LoadBalancerWeightAnnotationPrefix + "/" + forClusterName
```

Comment:
The name part of the annotation key cannot have a [slash](load-balancer). So we can use '-' or '.'. Also the name is limited to 63 chars which would be an issue for a cluster name longer than `62 - len(lhconstants.LoadBalancerWeightAnnotationPrefix)`.

```suggestion
	weightKey := lhconstants.LoadBalancerWeightAnnotationPrefix + "." + forClusterName
```

Reply:
I'm not sure I follow, maybe I missed something.
But according to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
I can have an optional DNS like prefix for the key (up to 255 characters) hence the `lhconstants.LoadBalancerWeightAnnotationPrefix = "lighthouse-load-balancer.submariner.io/weight"` Maybe I should change it to `lhconstants.LoadBalancerWeightAnnotationPrefix = "lighthouse-load-balancer-weight.submariner.io"` 
And then construct `prefix + "/" + clusterName`?

Then the annotations will be of
```
lighthouse-load-balancer-weight.submariner.io/cluster1: 10,
lighthouse-load-balancer-weight.submariner.io/cluster2: 80,
lighthouse-load-balancer-weight.submariner.io/cluster3: 10,
```

Reply:
I just realized you are correctly using the "/" separator so ignore that part of my comment.

> When you refer to the 63 characters limit - it is not clear in the documentations if it includes the prefix...

It does not include the prefix. It states that the "_name segment is required and must be 63 characters or less_" and "_prefix must...not longer than 253 characters in total_". 

You could use the form `lighthouse-load-balancer-weight.submariner.io/cluster1`.  That would give us more chars for the name part but it would still be an issue if a cluster name exceeded 63. But that's probably not likely in a production environment.

Reply:
I'd recommened even shortening it further to `lighthouse-lb-weight.submariner.io/clusterX`. Shorter the prefix we use, larger the clustername we can support before someone runs into error. Don't underestimate possibility of long clusternames with something like `country-state-city-datacenter-department-production-name`

Reply:
FWIU from the k8s docs Iinked, the max prefix length (253) is separate from the max name length (63).